crates/sui-framework/packages/sui-framework/sources/dynamic_object_field.move

```diff
@@ -110,4 +95,105 @@ module sui::dynamic_object_field {
         let (_field, value_addr) = field::field_info<Wrapper<Name>>(object, key);
         option::some(value_addr.to_id())
     }
+
+    public(package) fun internal_add<Name: copy + drop + store, Value: key>(
```

Comment:
Nit: it took me a second to realize what these were doing different from the (more-public) versions. It may be worth writing a comment or something? (Or bikeshedding as `add_unstorable`...)

Reply:
The name here is assuming a new feature, namely `internal` type parameters

So it would be 
```
public fun internal_add<Name: copy + drop + store, internal Value: key>(
                                                   ^^^^^^^^ new thing here
```

Reply:
With that context, thoughts?